src/main/java/Ui.java

```diff
@@ -0,0 +1,152 @@
+import java.util.Scanner;
+
+/**
+ * Text Ui of Duke.
+ */
+public class Ui {
+    private Scanner reader;
+
+    public Ui() {
+        this.reader = new Scanner(System.in);
+    }
+
+    /**
+     * Gets user input.
+     *
+     * @return String of next user input.
+     */
+    public String getInput() {
+        return reader.nextLine();
+    }
+
+    /**
+     * Shows line break.
+     */
+    public void printBreak() {
+        System.out.println("    ______________________________________________________________");
+    }
+
+    /**
+     * Shows Duke logo.
+     */
+    public void printLogo() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        printBreak();
+    }
+
+    /**
+     * Shows greeting message.
+     */
+    public void printGreet() {
+        System.out.println("    Hello! I'm Duke");
+        System.out.println("    What can I do for you?");
+        printBreak();
+    }
+
+    /**
+     * Shows all tasks in the list.
+     *
+     * @param tasks Task list.
+     */
+    public void list(TaskList tasks) {
+        printBreak();
```

Comment:
Maybe you can abstract away the printBreak()s in a function called format() so that you don't have to called printBreak in all the functions